integrationTests/vm/arwen/arwenVM_test.go

```diff
@@ -0,0 +1,235 @@
+package arwen
+
+import (
+	"encoding/hex"
+	"fmt"
+	"io/ioutil"
+	"math/big"
+	"testing"
+	"time"
+
+	"github.com/ElrondNetwork/elrond-go/data/transaction"
+	"github.com/ElrondNetwork/elrond-go/integrationTests/vm"
+	"github.com/ElrondNetwork/elrond-go/process/factory"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestVmDeployWithTransferAndGasShouldDeploySCCode(t *testing.T) {
+	senderAddressBytes := []byte("12345678901234567890123456789012")
+	senderNonce := uint64(0)
+	senderBalance := big.NewInt(100000000)
+	round := uint64(444)
+	gasPrice := uint64(1)
+	gasLimit := uint64(100000)
+	transferOnCalls := big.NewInt(50)
+
+	scCode, err := ioutil.ReadFile("./fib_arwen.wasm")
+	assert.Nil(t, err)
+
+	scCodeString := hex.EncodeToString(scCode)
+
+	tx := vm.CreateTx(
+		t,
+		senderAddressBytes,
+		vm.CreateEmptyAddress().Bytes(),
+		senderNonce,
+		transferOnCalls,
+		gasPrice,
+		gasLimit,
+		scCodeString+"@"+hex.EncodeToString(factory.ArwenVirtualMachine),
+	)
+
+	txProc, accnts, _ := vm.CreatePreparedTxProcessorAndAccountsWithVMs(t, senderNonce, senderAddressBytes, senderBalance)
+
+	err = txProc.ProcessTransaction(tx, round)
+	assert.Nil(t, err)
+
+	_, err = accnts.Commit()
+	assert.Nil(t, err)
+
+	expectedBalance := big.NewInt(99999700)
+	fmt.Printf("%s \n", hex.EncodeToString(expectedBalance.Bytes()))
+
+	vm.TestAccount(
+		t,
+		accnts,
+		senderAddressBytes,
+		senderNonce+1,
+		expectedBalance)
+}
+
+func Benchmark_VmDeployWithFibbonacciAndExecute(b *testing.B) {
```

Comment:
We should decide how to change these tests-with-manual-evaluation to something else (not benchmarks nor tests).

Reply:
done